test/specs/exporters/task-metrics/calc-mean-times.js

```diff
@@ -63,20 +78,46 @@ describe('Task Metrics', () => {
       const stats = {
         pplApplication: {
           submitToActionDays: [],
+          resubmitToActionDays: [],
           assignToActionDays: [5, 4, 3, 2, 1]
         }
       };
 
       const expected = {
         pplApplication: {
-          submitToActionDays: '-',
-          assignToActionDays: 3
+          submitToActionDaysMean: '-',
+          submitToActionDaysMedian: '-',
+          resubmitToActionDaysMean: '-',
+          resubmitToActionDaysMedian: '-',
+          assignToActionDaysMean: 3,
+          assignToActionDaysMedian: 3
         }
       };
 
       assert.deepEqual(calcMeanTimes(stats), expected);
     });
 
+    it('calculates the median values correctly for odd and even length arrays and empty arrays', () => {
+      const stats = {
+        pplApplication: {
+          submitToActionDays: [5, 2, 10, 9, 30],
+          resubmitToActionDays: [2, 6, 1, 8],
+          assignToActionDays: []
+        }
+      };
+      const output = calcMeanTimes(stats);
```

Comment:
might wanna rename this function & file to `calcAverageTimes` now that it also does median.